docroot/themes/contrib/civictheme/civictheme_library/components/00-base/collapsible/collapsible.js

```diff
@@ -257,7 +257,7 @@ CivicThemeCollapsible.prototype.collapse = function (animate, evt) {
   }
 
   if (evt && evt.target) {
-    if (evt.currentTarget !== t.el) {
+    if (evt.target !== t.el) {
```

Comment:
@govindmaloo Please review this change. 
https://github.com/salsadigitalauorg/civictheme_source/pull/878/files#diff-b6e4bf49e8b1f147cec7776e04f0565dee546bcef7dfe529f5686b5daf7cd924R260

This update changes the functionality of how collapsable works. 
It will close parent.